src/test/java/uk/ac/ebi/eva/contigalias/controller/ContigAliasControllerIntegrationTest.java

```diff
@@ -170,4 +175,96 @@ void test404NotFound() throws Exception {
         }
 
     }
+
+    @Nested
+    class AliasServiceTests {
```

Comment:
these tests are basically redundant with the ones in ContigAliasControllerTest.java , we can drop these, and keep the integration tests only for things that can't be tested in the unit tests.

Reply:
Created new issue #49